tools/fuzzer/fuzzer.h

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
@ranj063 Sorry for being annoying, but in .h files we (and kernel also) use C-style comments for license info. `/**/` instead of `//`.

Reply:
@jajanusz not annoying at all. consistency is important. Fixed all your comments now. Thanks!